plugins/org.eclipse.n4js.external.libraries/shipped-code/runtime/n4js-node/node_modules/encoding/package.json

```diff
@@ -17,9 +17,6 @@
   "license" : "MIT",
```

Comment:
Just for my understanding: Are these package.json generated?

Reply:
not generated; they are part of the npm packages used by "n4js-node"